src/stan-users-guide/latent-discrete.qmd

```diff
@@ -1063,7 +1063,7 @@ $$
 $$
 This estimator is justified by the law of iterated expectation, the fact that
 $$
-\mathbb{E}[h(\Theta)] = \mathbb{E}[\mathbb{E}[g(\Theta, Z] \mid \Theta)] = \mathbb{E}[g(\Theta, Z)] = \mu.
+\mathbb{E}[h(\Theta)] = \mathbb{E}[\mathbb{E}[g(\Theta, Z)] \mid \Theta] = \mathbb{E}[g(\Theta, Z)] = \mu.
```

Comment:
There's still a stray close square bracket on 1066 after `Z)`.

Reply:
Are you sure? it's that close square bracket close the inner Expectation?

Reply:
Oops.  You're right.  Thanks for your patience. As you can see, I have a bit of an issue with paren balancing---I usually just let emacs tell me :-).

Reply:
I cut-paste it into emacs with rainbow delimiters just to triple-check 😅